src/index.html

```diff
@@ -12,7 +12,50 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
You missed empty lines between sibling blocks (between every stars--N and the next one), you can check [Code Style Rules](https://github.com/mate-academy/layout_stars/blob/master/checklist.md).